src/object/cli_ec.c

```diff
@@ -29,6 +29,59 @@
 #define EC_TRACE(fmt, ...)
 #endif
 
+/* Move to next iov, it's caller's responsibility to ensure the idx boundary */
+#define daos_sgl_next_iov(iov_idx, iov_off)                                                        \
```

Comment:
Doesn't moving these out of `src/include` break backwards compatibility?

Reply:
Yes.